src/main.rs

```diff
@@ -15,7 +15,7 @@ use std::{env, io};
 use clap::{Arg, ArgAction, ArgGroup, ArgMatches, Command};
 use event_monitor::event;
 use libc::EFD_NONBLOCK;
-use log::{warn, LevelFilter};
+use log::{error, warn, LevelFilter};
```

Comment:
I don't see we call into `env_logger::init()` from the main function. Can you please take a look and see if that should be added so that logs are properly captured? Thanks.

Reply:
We could also extract the custom logger into a module and use the same logger for both bins. Consistency is king.

Reply:
@phip1611
Absolutely. An alternative to your suggestion is to use `env_logger` and changing [the target](https://github.com/rust-cli/env_logger/issues/125#issuecomment-1582209797) when the user specifies a log file. Of course there are [many other logger implementations to choose from](https://docs.rs/log/latest/log/#available-logging-implementations).

This would however change the output format of `cloud-hypervisor`, especially `std::thread::current().name()` would be missing with `env_logger`.

>  use the same logger for both bins

Actually it would make sense to use the same logger for all bins, right? So current users of `env_logger`, such as `vhost_user_net`.